.github/workflows/CI.yml

```diff
@@ -246,6 +246,7 @@ jobs:
 
     env:
       TEST_SUITES: testinstall
+      NO_COVERAGE: 1
```

Comment:
Why is this here?

Reply:
Ah! Thank you for noticing this @fingolfin. I can't remember.

But I realised that the `test-cygwin` job runs neither of the scripts `dev/ci-gather-coverage.sh` and `dev/ci-gather-coverage.sh`, so I thought: what's the point of tracking code coverage if we then don't do anything with it? And I thought that including `NO_COVERAGE: 1` (meaning `io` and `profiling` no longer need to be compiled) would speed up the Cygwin job, in which compilation is slow.

...But of course, such a change (or indeed the opposite, actually gathering and uploading the coverage data in Cygwin) should be in a separate PR 🙂.

Reply:
Removed.